src/commands/Social/mylevel.ts

```diff
@@ -18,8 +20,8 @@ export default class extends SkyraCommand {
 	}
 
 	public async run(message: KlasaMessage, [user = message.author]: [KlasaUser]) {
-		const memberSettings = await this.client.queries.fetchMemberSettings(message.guild!.id, user.id);
-		const memberPoints = memberSettings ? memberSettings.point_count : 0;
+		const memberSettings = await (await DbSet.connect()).members.findOne({ where: { userID: user.id, guildID: message.guild!.id }, cache: Time.Minute * 15 });
```

Comment:
Consistency with the rest of the files, please
```suggestion
		const { members } = await DbSet.connect();
		const memberSettings = await members.findOne({ where: { userID: user.id, guildID: message.guild!.id }, cache: Time.Minute * 15 });
```